website/pages/style.css

```diff
@@ -6,6 +6,19 @@
 @import '~@hashicorp/react-global-styles/_temporary-to-remove/layout.css';
 @import '~@hashicorp/react-global-styles/_temporary-to-remove/tables.css';
 
+@media print {
+  /* @todo: remove alongside ~@hashicorp/react-global-styles/_temporary-to-remove/layout.css */
+  .g-container {
+    max-width: 34em;
+  }
+
+  /* @todo: remove alongside ~@hashicorp/react-global-styles/_temporary-to-remove/tables.css */
+  table {
+    margin-bottom: 0;
+    margin-top: 20px;
+  }
+}
```

Comment:
These are used in a variety of ways outside of docs sites, I think it might be safer to include this batch with the "general purpose docs site overrides" section